acceptance/tests/windows/PA-2191_windows_nocodepage_utf8_fallback.rb

```diff
@@ -0,0 +1,21 @@
+test_name 'PA-2191 - winruby fallsback to UTF8 for invalid CodePage' do
+  confine :to, platform: 'windows'
+
+  tag 'audit:medium',
+      'audit:acceptance'
+
+  initial_chcp_code = on(agent, 'cmd.exe /c chcp').stdout.delete("^0-9")
```

Comment:
👍 I verified the test correctly fails when running with a puppet version that doesn't contain the fix.

One nit though, using `agent` can be dangerous due to https://tickets.puppetlabs.com/browse/BKR-558. If you have a single Windows agent, then everything will work fine. If you have multiple Windows agents, and pass `agent` to a method that doesn't handle Arrays, it will bomb unexpectedly. I find it better to use always to avoid confusion:

```ruby
agents.each do |host|
  ... on(host, ...)
end
```

Reply:
Updated, thanks for the clarification on that.